understanding/page-titled.html

```diff
@@ -58,121 +56,94 @@
                   <p>
-                     <a href="#dfn-web-page">Web pages</a> have titles that describe topic or purpose.
+                     <a href="#dfn-web-page">ウェブページ</a>には、主題又は目的を説明したタイトルがある。
                      
                   </p>
                </div>
             </aside>
             <div class="excol-all"></div>
             <section id="brief">
-               <h2>In Brief</h2>
+               <h2>要約</h2>
                <dl>
                   
-                  <dt>Goal</dt>
-                  <dd>Each web page has a meaningful title.</dd>
+                  <dt>目標</dt>
+                  <dd>各ウェブページに、意味のあるタイトルを付ける。</dd>
                   
-                  <dt>What to do</dt>
-                  <dd>Provide a descriptive page title using appropriate technology.</dd>
+                  <dt>何をすればよいか</dt>
+                  <dd>適切な技術を用いて、説明的なページタイトルを付ける。</dd>
                   
-                  <dt>Why it's important</dt>
-                  <dd>Page titles help users identify and distinguish different pages.</dd> 
+                  <dt>なぜそれが重要か</dt>
+                  <dd>ページタイトルによって、利用者が異なるページを識別、及び区別するのに役立つ。</dd> 
                   
                </dl>
             </section>
             <section id="intent">
-               <h2>Intent</h2>
-               <p>The intent of this Success Criterion is to help users find content and orient themselves
-                  within it by ensuring that each Web page has a descriptive title. Titles identify
-                  the current location without requiring users to read or interpret page content. When
-                  titles appear in site maps or lists of search results, users can more quickly identify
-                  the content they need. User agents make the title of the page easily available to
-                  the user for identifying the page. For instance, a user agent may display the page
-                  title in the  window title bar or as the name of the tab containing the page.
+               <h2>意図</h2>
+               <p>この達成基準の意図は、各ウェブページにその内容を示すページタイトルを付けることによって、利用者がコンテンツを見つけやすく、自分の現在位置を確認しやすくすることである。タイトルは、ページのコンテンツを読んだり解釈したりすることを利用者に要求することなしに、現在位置を特定する。タイトルがサイトマップ又は検索結果のリストに表示されたときに、利用者は自分の求めているコンテンツをより素早く確認できるようになる。ユーザエージェントは、利用者がそのページを確認できるように、ページのタイトルを容易に見つけられるようにする。例えば、ユーザエージェントは、そのページタイトルをウィンドウのタイトルバーに表示するか、又はそのページを表示しているタブの名前として表示する。
                   
                </p>
-               <p>In cases where the page is a document or a web application, the name of the document
-                  or web application would be sufficient to describe the purpose of the page. Note that
-                  it is not required to use the name of the document or web application; other things
-                  may also describe the purpose or the topic of the page. 
+               <p>ページが文書又はウェブアプリケーションの場合、文書又はウェブアプリケーションの名称はページの目的を説明するのに十分だろう。ここで注意すべきなのは、文書又はウェブアプリケーションの名称を使用することを求められていないことである。他の要素によってページの目的又は主題が説明される場合もあるだろう。
```

Comment:
最初の文は要するに「名前をタイトルに使えばいいじゃん」という話なのですが、すこしわかりにくい感じがします。こんな感じにしてはどうでしょうか。

```suggestion
               <p>ページが文書又はウェブアプリケーションの場合、その文書又はウェブアプリケーションの名称が、ページの目的を説明するのに十分なものとなるだろう。ここで注意すべきなのは、文書又はウェブアプリケーションの名称を使用することが必須ではないことである。他の要素によってページの目的又は主題が説明される場合もあるだろう。
```